docs/web-wallet.md

```diff
@@ -0,0 +1,249 @@
+7. **Verification Completion**: The user finally clicks the ‘Continue’ button, upon which they are redirected to the application where the verification can be completed and the application flow can continue.
+
+:::note
+
+For credentials that do not support embedded issuance directly through the Web Wallet, applications can still facilitate a seamless verification process. Users who have previously issued credentials stored on their mobile devices can continue the verification from their mobile device. To do this, users can simply click the “Verify from Mobile” button in the Web Wallet interface. This action seamlessly integrates with the existing credentials on their device, allowing them to continue with the verification flow without interruption.
+
+:::
+
+## Integration Process for the Verifiers
+
+### Step 1 - Choosing Your Credential Schema
+
+Select the credential schema that aligns with your requirements from our [Schema Explorer](https://schema-builder.polygonid.me/) or any other source.
+Standardized schemas, such as KYC, are available. Alternatively, you can create a custom schema using our [Schema Builder](https://schema-builder.polygonid.me/builder).
+
+### Step 2 - Creating the Query
+
+Define what information you need from your users and build a query using the [Query Builder](https://schema-builder.polygonid.me/query-builder).
+Once you create the query, you can select the network and test your query.
+Copy the generated code snippet of the query for later integration into your verifier setup.
+
+### Step 3 - Setting up the verifier
+
+The verifier can be setup in your application using the [Verifier SDK](./verifier/verifier-overview.md#verifier-sdk) or the [JS SDK](./js-sdk/js-sdk-components/proof.md#verify-zero-knowledge-proof-using-verifyproof-method).
+The verification can be performed in two ways:
+
+- [Off-Chain verification](./verifier/verification-library/verifier-library-intro)
+- [On-Chain verification](./verifier/on-chain-verification/overview)
+
+:::note
+
+The server setup for Off-Chain Verification and the Smart Contract setup for On-Chain Verification, as detailed in the provided links, remain unchanged. The only modification required is in the client side of the application. Previously, proof requests could only be shared via embedded QR codes. Now, verifiers have a new, more efficient method for requesting proofs from users by leveraging the Web Wallet.
+
+:::
+
+:::note
+
+In Case of Off-Chain Verification, once the user goes through the flow of verification in the Web Wallet, the zero-knowledge proof is returned as a JWZ token in a callback to your verifier backend. You need to handle this callback coming from the Web Wallet and verify the proof.
```

Comment:
We could link to the JWZ docs [here](http://localhost:3000/docs/wallet/wallet-sdk/polygonid-sdk/iden3comm/jwz).

Reply:
added